src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/loadbalancer/ElasticLoadBalancerFormula.kt

```diff
@@ -50,4 +51,9 @@ class ElasticLoadBalancerFormula : LoadBalancerFormula {
             resource = stack
         )
     }
+
+    override fun updateJiraConfiguration(
+        ssh: Ssh,
+        unpackedProduct: String) {
```

Comment:
Formatting here. `) {` should be in the next line.

Reply:
Done.